roles/dcos_agent_windows/defaults/main.yml

```diff
@@ -0,0 +1 @@
+---
```

Comment:
don't think we should have empty files here. If we do not need it lets just delete it

Reply:
agreed